guides/common/modules/proc_running-a-scan-on-demand.adoc

```diff
@@ -0,0 +1,25 @@
+[id="running-a-scan-on-demand_{context}"]
+= Running a Scan on Demand
```

Comment:
Should this be "Running an OpenSCAP Scan on Demand"? Are there any other types of scans? Or should we say "Run a security compliance scan?" as a more generic way? Just some ideas ...